src/services/formatters/YscopeFormatter/FieldFormatters/RoundFormatter.ts

```diff
@@ -0,0 +1,29 @@
+import {Nullable} from "../../../../typings/common";
+import {YscopeFieldFormatter} from "../../../../typings/formatters";
+import {JsonValue} from "../../../../typings/js";
+import {jsonValueToString} from "../utils";
+
+
+/**
+ * A field formatter that rounds numerical values to the nearest integer.
+ * For non-numerical values, the field's value is converted to a string then returned as-is.
+ * Options: none.
```

Comment:
Sorry for missing this in the sugguestion.
```suggestion
 * Options: None.
```